### PR TITLE
add missing options: PUBLICKEY_ACCEPTED_TYPES, PROCESS_CONFIG, ...

### PR DESCRIPTION
**Pull request by enkore**
(eref 42d478fb6976)

### CODE REVIEW

**Comment by enkore** [comment]
CI failure seems to be a Debian mirror problem, because I can download the 404'ing URL with no issues:

> E: Failed to fetch http://deb.debian.org/debian/pool/main/c/cmake/cmake-data_3.18.4-2_all.deb  404  Not Found [IP: 146.75.34.132 80]

**Comment by enkore** [comment]
rebased for CI fix

File: ssh/options.c [hunk 1]
```diff
@@ -1,14 +1,16 @@
-/* Generated by Cython 0.29.21 */
+/* Generated by Cython 0.29.26 */
```
Comment (pkittenis) [inline ssh/options.c:1]: Going to need to re-generate all the *.c files when changing Cython versions. `rm -f ssh/*.c; python setup.py build_ext -i`

**Review by pkittenis — CHANGES_REQUESTED** [review]

File: ssh/options.c (hunk 1 above)
Comment (enkore) [reply ssh/options.c]: I've regenerated it with the committed version for this PR to keep changes local and make another one for bumping the Cython version

**Comment by pkittenis** [comment]
Thanks for making these changes. Have pulled this into #49 and re-generated sources with latest cython.

**Pull request closed without merging**